content/insiders/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "LocalStack Insiders"
+foo: "foobar"
+description: "Join the LocalStack Insiders program to help us shape the future of LocalStack! Test drive new features and engage in conversations with us so we can learn from your specific use cases."
+date: 2023-07-11T08:38:36Z
+lastmod: 2023-07-11T08:38:03Z
+draft: false
+images: []
+---
+
+### Fellow Developers!
```

Comment:
Did we had a chance to look at rephrasing this? 🤔

Reply:
yes, but no good alternative was proposed